integration-cli/docker_cli_ps_test.go

```diff
@@ -965,3 +965,22 @@ func (s *DockerSuite) TestPsListContainersFilterPorts(c *check.C) {
 	c.Assert(strings.TrimSpace(out), checker.Not(checker.Equals), id1)
 	c.Assert(strings.TrimSpace(out), checker.Equals, id2)
 }
+
+func (s *DockerSuite) TestPsNotShowLinknamesOfDeletedContainer(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+
+	dockerCmd(c, "run", "-d", "--name=aaa", "busybox", "top")
+	dockerCmd(c, "run", "-d", "--name=bbb", "--link=aaa", "busybox", "top")
```

Comment:
Do the containers have to be running for this test, or is `docker create` sufficient?